### DISCUSSION TITLE
New version?

### DISCUSSION THREAD

**Discussion by PapsDizzy** [Jyers/Marlin#1495] [Q&A]
Hi all, there hasn't been a new jyers version lately. when will there be a version 1.3.6 or maybe even 2.0? I love this firmware, but version 1.3.4 was great, 1.3.5 and 1.3.5b are also great but apparently not as stable as 1.3.4. therefore I ask if there might be a new version soon that combines the stability of version 1.3.4 with the fixes and updates of version 1.3.5 / b. please excuse my english i used the google translator.

**Comment by malnaxdr** [comment]
Hi to everybody. I'm agree about the version 1.4.5b is good but in my Ender 3 V2 there is a bug on calculation of time elapsed and time remaining. Cura tell me 50 minutes for a project and during printing the time remaining start from 1:30 minutes, moreover the time elapsed is wrong. I don't know if you have found the same problem. Enjoy

**Reply by malnaxdr** [reply]
Thank you for your feedback
